electron-app/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "private": true,
+  "name": "electron-app",
+  "version": "1.0.2",
+  "dependencies": {
+    "@theia/core": "latest",
+    "@theia/editor": "latest",
+    "@theia/filesystem": "latest",
+    "@theia/languages": "latest",
+    "@theia/markers": "latest",
+    "@theia/messages": "latest",
+    "@theia/monaco": "latest",
+    "@theia/navigator": "latest",
+    "@theia/preferences": "latest",
+    "@theia/process": "latest",
+    "@theia/terminal": "latest",
+    "@theia/typescript": "latest",
+    "@theia/workspace": "latest",
+    "sunix-che-theia-factory-frontend-extension": "^1.0.2"
```

Comment:
something wrong there ?

Reply:
ok let's remove the electron app what do you think ?